website/docs/contributing/general/ADR/domain-language.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "ADR: Domain language"
+---
+
+## Background
+
+In the codebase, we have seen the need to define a domain language that we use to refer to features, methods to keep it consistent across the codebase. This ADR will contain a growing list of domain language used to keep the consistency across the codebase.
+
+## Decision
+
+We have decided to use the same domain language for the features we develop. Each feature will have it's own domain language to keep it consistent across the codebase.
+
+## Change requests domain language
+
+* *Change request*
+
+An entity referring to the overarching data structure of a change request. A change request contains changes, and can be approved or rejected.
+
+* *Changes*
+
+An entity referring to a single change within a change request
```

Comment:
Is this correct? Isn't "changes" a "list of changes" and not a single change? This is unclear to me. Also, what is "an entity" in this context?